src/Response/GetPayload/PayloadMeta.php

```diff
@@ -19,11 +19,13 @@ public function __construct(
         public readonly bool $pushed,
         public readonly bool $appOpened,
         public readonly bool $isXapp,
+        public readonly bool $pathfinding,
```

Comment:
This is what the acceptance tests are complaining about now. Do `pathfinding` and `pathfindingFallback` have to be provided or can they have a default? @WietseWind are these always present on a `get`?

Reply:
They will always return `false` on a `GET`, even if you provide `null` on a `POST`.